README.md

```diff
@@ -84,9 +84,10 @@ Head over to [asyncapi/raml-dt-schema-parser](https://www.github.com/asyncapi/ra
 1. Run tests with `npm test`
 1. Write code and tests.
 1. Make sure all tests pass `npm test`
-1. Generate new API docs `npm run docs`
 1. Update bundle for client-side parser `npm run bundle`
```

Comment:
Cant we include this in the workflow as well?

Reply:
release one? it is already there, or?

Reply:
sorry, the `Update bundle for client-side parser `npm run bundle``

Reply:
Then we can probably delete this line too, right?

Reply:
Yea, it can be removed https://github.com/asyncapi/parser-js/blob/e5ffb4a01c5932c180aac969df29edfceda0253a/.github/workflows/release.yml#L23

Reply:
yes, you are right, removing this line 
`npm run prepublishOnly` runs in release pipeline and it runs `npm run bundle`